xml/art_solution_automation.xml

```diff
@@ -0,0 +1,438 @@
+        <para>Database instance (this adds the required users, tables, views, etc to the current Hana database)</para>
+      </listitem>
+    </itemizedlist>
+    <para>
+      The &salt; formula can be used in these scenarios:
+    </para>
+    <itemizedlist>
+      <title>Scenarios for <package>sapnwbootstrap-formula</package></title>
+      <listitem>
+        <para>Enqueue replication server version 1</para>
+      </listitem>
+      <listitem>
+        <para>Enqueue replication server version 2</para>
+      </listitem>
+      <listitem>
+        <para>Azure, AWS, and GCP automated adjustments for each provider best-practices</para>
+      </listitem>
+    </itemizedlist>
+    <example>
+      <title>Installation</title>
+      <screen>zypper addrepo --refresh obs://network:ha-clustering:Factory ha-clustering
+zypper install --repo ha-clustering sapnwbootstrap-formula</screen>
+    </example>
+
+    <itemizedlist>
+      <title>Best Practises</title>
+      <listitem>
+        <para><link xlink:href="https://documentation.suse.com/sbp/all/html/SAP_HA740_SetupGuide_AWS/index.html"
+          >&netweaver; &ha; Cluster 7.40 for the AWS Cloud - Setup Guide (Code 12)</link></para>
+      </listitem>
+      <listitem>
+        <para><link xlink:href="https://www.suse.com/media/white-paper/sap_netweaver_availability_cluster_740_setup_guide.pdf"
+          >&netweaver; &ha; Cluster 7.40 - Setup Guide (Code 12)</link></para>
+      </listitem>
+      <listitem>
+        <para><link xlink:href="https://www.suse.com/media/white-paper/sap_netweaver_availability_cluster_740_setup_guide.pdf"
+          >Simple Stack - &netweaver; &ha; on &sle; 12 (SP1 or newer)</link></para>
+      </listitem>
+      <listitem>
+        <para><link xlink:href="https://documentation.suse.com/sbp/all/html/SAP_S4HA10_SetupGuide-SLE12/index.html"
+          >&s4h; - Enqueue Replication 2 &ha; Cluster - Setup Guide</link></para>
+      </listitem>
+      <listitem>
+        <para><link xlink:href="https://docs.microsoft.com/en-us/azure/virtual-machines/workloads/sap/high-availability-guide-suse"
+          >Azure</link></para>
+      </listitem>
+    </itemizedlist>
+  </sect1>
+
+  <sect1 xml:id="sec-sol-autom-ha-cluster-config">
+    <title>&ha; Cluster Automated Configuration</title>
+    <para>
+      &salt; formula <package>habootstrap-formula</package> for bootstrapping and managing
+      a high availability cluster using &crmshell;.
+    </para>
+    <formalpara>
+      <title>Link:</title>
+      <para>See <link xlink:href="https://github.com/SUSE/habootstrap-formula"/>.</para>
+    </formalpara>
+    <itemizedlist>
+      <title>Properties for <package>habootstrap-formula</package></title>
+      <listitem>
+        <para>
+          Rund on premises and on Cloud service providers (Azure, GCP, AWS).
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          Only passwordless GPG keys are supported. Already existing keys cannot be used.
```

Comment:
I don't really understand this point. The habootstrap-formula doesn't interact in any case with GPC keys. Maybe you are talking about `ssh` keys... But I wouldn't understand the sentence neither hehe

Reply:
I extracted this from the README and Confluence page. :wink:  If this is wrong, then we need to change it in these places too.

Reply:
Ok, I see. The formulas docs talks about how to encrypt the pillar files, something that is a general thing about salt. I wouldn't even comment that here.
But anyway, you are right, the Github repository README is a big mess and it needs a total refactoring

Reply:
I removed this item if you think this is a more "general thing about Salt".

> But anyway, you are right, the Github repository README is a big mess and it needs a total refactoring.

I would really help here, but I don't know enough to improve it.

The only tip I can give and I think that is really important: **make it consistent**. Try to apply the same structure in every repo. That makes it easier to follow and absorb the content. :wink:  I can help with the structuring, other need to include the content.

Reply:
@tomschr I have create a task for our team to update the docs. We will work on that this and the next week

Reply:
Thank you Xabier. :+1: